nix/postgres/setup-script.nix

```diff
@@ -1,5 +1,27 @@
 { config, pkgs, lib }:
 let
+  setupInitialSchema = dbName: schema: ''
+    ${lib.optionalString (schema != null) ''
+    echo "Applying database schema on ${dbName}"
+    if [ -f "${schema}" ]
+    then
+      echo "Running file ${schema}"
+      awk 'NF' "${schema}" | psql -d ${dbName}
```

Comment:
This uses `/usr/bin/awk` on macOS? I'd think `lib.getExe pkgs.gawk` would be more reproducible.

Reply:
It is actually run inside a script that gets its runtime inputs from [here](https://github.com/juspay/services-flake/blob/2211042f3e04de6b105f8d70c0be6213a2127d88/nix/postgres/setup-script.nix#L86)